packages/keyboard/setup.py

```diff
@@ -0,0 +1,75 @@
+"""pi-top Python SDK."""
+
+import os
+import sys
+
+from setuptools import setup
+
+if not sys.version_info >= (3, 7):
+    raise ValueError("This package requires Python 3.7 or above")
+
+HERE = os.path.abspath(os.path.dirname(__file__))
+
+__version__ = os.environ.get("PYTHON_PACKAGE_VERSION", "0.0.1.dev1")
+assert __version__ != ""
+
+__project__ = "pitop.keyboard"
+__author__ = "pi-top"
+__author_email__ = "deb-maintainers@pi-top.com"
+
+__url__ = "https://github.com/pi-top/pi-top-Python-SDK"
+__platforms__ = "ALL"
+
+# https://pypi.org/classifiers/
+__classifiers__ = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Education",
+    "Intended Audience :: Developers",
+    "Topic :: Education",
+    "Topic :: System :: Hardware",
+    "License :: OSI Approved :: Apache Software License",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: Implementation :: PyPy",
+]
+
+__keywords__ = [
+    "pi-top",
+    "raspberrypi",
+    "battery",
+]
+
+__requires__ = [
+    f"pitop.common=={__version__}",
```

Comment:
I think this needs to include XLib, the vendored pynput package imports it:
```
Traceback (most recent call last):
  File "/home/pi/subpackages/test.py", line 3, in <module>
    button = KeyboardButton("a")
  File "/home/pi/subpackages/.virtualenv/lib/python3.9/site-packages/pitop/keyboard/keyboard_button.py", line 18, in __init__
    from .vendor.pynput.keyboard import Listener
  File "/home/pi/subpackages/.virtualenv/lib/python3.9/site-packages/pitop/keyboard/vendor/pynput/__init__.py", line 22, in <module>
    from . import keyboard, mouse
  File "/home/pi/subpackages/.virtualenv/lib/python3.9/site-packages/pitop/keyboard/vendor/pynput/keyboard/__init__.py", line 48, in <module>
    from ._xorg import Controller, Key, KeyCode, Listener
  File "/home/pi/subpackages/.virtualenv/lib/python3.9/site-packages/pitop/keyboard/vendor/pynput/keyboard/_xorg.py", line 28, in <module>
    import Xlib.display
ModuleNotFoundError: No module named 'Xlib'
```

Reply:
added `xlib` as a dependency to the `keyboard`  subpackage